### PR TITLE
.NET 5.0 platform

### PR DESCRIPTION
**Pull request by BeatSkip**
- Added conditional SDK selection
	(As .NET 5 uses 'Microsoft.NET.Sdk' instead
	of 'Microsoft.NET.Sdk.WindowsDesktop')
- Added .NET 5.0 Package includes

currently validating everything
but yet to see any problems

### CODE REVIEW

**Comment by Roemer** [comment]
Awesome thank you. I will also do some checks soon so we can merge this change and do a new release.

**Comment by BeatSkip** [comment]
i updated the versioning and cleaned the packagereferences to the newer format that you already used for the rest. will run functional tests today in a personal project completely built on .net 5.0

**Comment by sharpordie** [comment]
Do you think we can soon use FlaUI with .NET5?

**Comment by BeatSkip** [comment]
I am actually using it on .net 5.0 currently. The project isn't finalized yet, but everything seems to work just fine @sharpordie

File: src/FlaUI.Core/FlaUI.Core.csproj [hunk 1]
```diff
@@ -1,15 +1,17 @@
-﻿<Project Sdk="Microsoft.NET.Sdk.WindowsDesktop">
-
+﻿<Project>
+  <Sdk Condition="'$(TargetFramework)' != 'net5.0-windows'" Name="Microsoft.NET.Sdk.WindowsDesktop" />
```
Comment (Roemer) [inline src/FlaUI.Core/FlaUI.Core.csproj:2]: Unfortunately this does not work as this adds the imports before everything else. Fortunately it is allowed to use `Microsoft.NET.Sdk.WindowsDesktop` also in .net 5 and there is no warning shown anymore. See https://github.com/dotnet/sdk/pull/13441

File: src/FlaUI.Core/FlaUI.Core.csproj [hunk 2]
```diff
@@ -33,6 +35,11 @@
     <IntermediateOutputPath>obj\$(Configuration)\Signed</IntermediateOutputPath>
   </PropertyGroup>
 
+  <PropertyGroup>
```
Comment (Roemer) [inline src/FlaUI.Core/FlaUI.Core.csproj:38]: Why was this added?

File: src/FlaUI.UIA2/FlaUI.UIA2.csproj [hunk 3]
```diff
@@ -37,7 +39,7 @@
     <ProjectReference Include="..\FlaUI.Core\FlaUI.Core.csproj" />
   </ItemGroup>
 
-  <ItemGroup Condition="'$(TargetFramework)' != 'netcoreapp3.0' Or '$(TargetFramework)' != 'netcoreapp3.1'">
+  <ItemGroup Condition="'$(TargetFramework)' != 'netcoreapp3.0' And '$(TargetFramework)' != 'netcoreapp3.1'">
```
Comment (Roemer) [inline src/FlaUI.UIA2/FlaUI.UIA2.csproj:42]: Good catch ;) That removes some build warnings

**Comment by Roemer** [comment]
I made a new commit on your behalf which should work fine: https://github.com/FlaUI/FlaUI/commit/8d736cbd36bb6b6e1957ee7765f3ab852c520326
So this PR should be obsolete. Thanks for your help!
